### PR TITLE
Kbrake feature osquery snapshot

### PR DESCRIPTION
**Pull request by elastickent**
Added support for [snapshots](https://osquery.readthedocs.io/en/stable/deployment/logging/#snapshot-format) to osquery filebeat module.

### CODE REVIEW

**Comment by ruflin** [comment]
@elastickent Not sure why your email is not recognised by our CLA checker? https://www.elastic.co/contributor-agreement

**Comment by elastickent** [comment]
@ruflin This is my first PR at Elastic. My kent@elastic.co is the primary address for my GitHub account. So I'm not sure why it's prompting me to sign the CLA.

File: filebeat/filebeat.yml [hunk 1]
```diff
@@ -142,12 +142,18 @@ setup.kibana:
 
 #================================ Outputs =====================================
 
+# Console
```
Comment (ruflin) [inline filebeat/filebeat.yml]: This should not be commited. You need to run `make update` and push again.

**Comment by ruflin** [comment]
General note: In the future best clone the repository, create your branch there and open a PR instead of creating a branch in this repo. No need to change anything for this PR.

**Comment by jsoriano** [comment]
@elastickent could you add a changelog entry?

File: filebeat/filebeat.yml (hunk 1 above)
Comment (elastickent) [reply filebeat/filebeat.yml]: Seems I have gotten my local repo a bit out of sync and having trouble pushing it. I've tried to resolve them with manual updates, but let me know if you think I should just put my changes into a freshly branched PR off of the current master.

**Comment by kaiyan-sheng** [comment]
Seems like this PR is almost ready but haven't had updates in a while. @elastickent Could you take a look at this again please? Thanks!

**Comment by andresrc** [comment]
Closing due to lack of activity. Please reopen when continuing it. Thanks!

**Pull request closed without merging**